projects/sdk/src/lib/silo.ts

```diff
@@ -456,18 +483,25 @@ export class Silo {
    */
   async getPlantableSeeds(_account?: string) {
     const account = await Silo.sdk.getAccount(_account);
-    // TODO: this is wrong
-    return Silo.sdk.contracts.beanstalk.balanceOfEarnedSeeds(account).then((v) => Silo.sdk.tokens.SEEDS.fromBlockchain(v));
+    throw new Error("Not implemented");
   }
 
   /**
-   * Get a Farmer's Grown Stalk since last Mow.
-   * @param _account
-   * @returns
+   * Get a Farmer's Grown Stalk since last Mow. Aggregates Grown Stalk across
+   * all whitelisted tokens.
    */
   async getGrownStalk(_account?: string) {
     const account = await Silo.sdk.getAccount(_account);
-    return Silo.sdk.contracts.beanstalk.balanceOfGrownStalk(account).then((v) => Silo.sdk.tokens.STALK.fromBlockchain(v));
+
+    const results = await Promise.all(
+      Silo.sdk.tokens.siloWhitelistAddresses.map((address) => {
+        return Silo.sdk.contracts.beanstalk.balanceOfGrownStalk(account, address);
+      })
+    );
```

Comment:
Sensible default here was to do all whitelisted tokens, since the user probably wants to see all of their grown stalk in one number